.snippets/code/develop/toolkit/api-libraries/polkadot-js-sdk/send-txs.js

```diff
@@ -0,0 +1,10 @@
+// Assuming you have an `alice` keypair from the Keyring
+const RECIPIENT = 'RECIPIENT_ADDRESS';
+const AMOUNT = 12345; // Amount in the smallest unit (e.g., Planck for DOT)
```

Comment:
To be consistent, I think it's better to replace here with the following:

```suggestion
const RECIPIENT = 'INSERT_RECIPIENT_ADDRESS';
const AMOUNT = 'INSERT_VALUE'; // Amount in the smallest unit (e.g., Planck for DOT)
```